lib/autoload/sfAutoload.class.php

```diff
@@ -138,7 +138,15 @@ public function reloadClasses($force = false)
 
     $file = $configuration->getConfigCache()->checkConfig('config/autoload.yml');
 
-    $this->classes = include($file);
+    if (defined('HHVM_VERSION'))
+    {
+      // workaround for https://github.com/facebook/hhvm/issues/1447
+      $this->classes = eval(str_replace('<?php', '', file_get_contents($file)));
```

Comment:
I do not know any other way to disable it than the [disable_functions](http://php.net/manual/en/ini.core.php#ini.disable-functions) ini directive.